code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1234,3 +1250,39 @@
 	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, 0.5)
 	..()
 	. = 1
+
+/datum/reagent/medicine/badstims  //These are bad for combat on purpose. Used in adrenal implant.
+	name = "Experimental Stimulants"
+	description = "Experimental Stimulants designed to get you away from trouble."
+	reagent_state = LIQUID
+	color = "#F5F5F5"
+
+/datum/reagent/medicine/badstims/on_mob_life(mob/living/carbon/M)
+	..()
+	if(prob(30) && iscarbon(M))
+		var/obj/item/I = M.get_active_held_item()
+		if(I && M.dropItemToGround(I))
+			to_chat(M, "<span class='notice'>Your hands spaz out and you drop what you were holding!</span>")
+	if(prob(40))
+		M.losebreath++
+		M.adjustOxyLoss(1, 0)
+	M.adjustStaminaLoss(-10, 0)
+	M.adjustToxLoss(0.5, 0)
```

Comment:
I do agree this is a bit much though since

7.5 damage you can't heal out of unless you see medbay (or use puke juice which comes with side effects) on top of >15 oxy damage is very steep.

Reply:
I get where you are coming from however this is in line with how the better chems affect you, in this case the oxy (toned down the prob) and tox (left as is) damage is from aranesp. Ling adrenals also give you tox, so there is some precedent. If you think it's a deal breaker, we can adjust it.

Reply:
ah I see, it's just a lesser form of aran?

Reply:
The stamina and oxy/tox effects are, yeah. As I said, the tox damage is left as is, since it's pretty low to begin with. 

For reference, aranesp has:
M.adjustStaminaLoss(-18, 0)
M.adjustToxLoss(0.5, 0)
if(prob(50))
	M.losebreath++
	M.adjustOxyLoss(1, 0)